### PR TITLE
factor nearly done

### PR DESCRIPTION
**Pull request by Many5900**

### CODE REVIEW

File: report/chapters/problem_analysis.typ [hunk 1]
```diff
@@ -7,6 +7,29 @@
 == PACT
 
 == FACTOR
+We have applied the FACTOR model to the absence calendar system to ensure a comprehensive analysis of the key components and requirements needed for successful implementation. The FACTOR model provides a structured approach to identifying the system’s functionality, the domain in which it operates, the conditions of development, the technologies involved, the objects that are central to the system, and the overall responsibilities. By using this model, we aim to clearly define how our system will function within the organizational context of KMD.
```
Comment (tbdlarsen) [inline report/chapters/problem_analysis.typ:10]: Thought we agreed not to use we in the report, unsure though. Good section, maybe add that it also serves as a way to communicate exactly between KMD and us

File: report/chapters/problem_analysis.typ [hunk 2]
```diff
@@ -7,6 +7,29 @@
...
+
+=== A - (Application domain)
+The application falls under the domain of human resource management, specifically focusing on absence tracking and planning within an organization. It is designed to facilitate self-service absence management for employees while also providing oversight capabilities for managers. It supports both individual users who wish to manage their absences and managers who need to monitor the availability of their teams.
+
+=== C - (Conditions)
+The system is being developed by a team of bachelor software students in cooperation with KMD employees, who provide the necessary knowledge and information regarding company-specific requirements.
+
+Additionally, the system must be versatile enough to accommodate users with varying levels of IT experience. It will be accessible from a variety of devices, including desktops, tablets, and mobile phones, allowing users to engage with the system in the most convenient manner possible.
```
Comment (tbdlarsen) [inline report/chapters/problem_analysis.typ:21]: A bit unclear, that i will be available through a web application in this section
Never mind, its right underneath 🫡

File: report/chapters/problem_analysis.typ [hunk 3]
```diff
@@ -7,6 +7,29 @@
...
+
+=== T - (Technology)
+The system will be developed as a web application, with a client-side interface that runs on each user's device and communicates with a server over a wireless network. The server will host the system's logic and maintain the database. The backend application will utilize a RESTful API, implemented in Java using the popular framework "spring boot", while the frontend will be developed using frameworks like React to ensure a responsive and user-friendly interface. Data concerning employees, absences, and teams will be managed through a relational database named PostgreSQL.
```
Comment (tbdlarsen) [inline report/chapters/problem_analysis.typ:24]: Maybe add that the network that the application runs on is a protected one.

**Review by Dandan2208 — APPROVED** [review]
🍓

**Pull request merged**